omod/src/main/webapp/pages/metadata/roles/manageRoles.gsp

```diff
@@ -1,67 +1,49 @@
 <%
     ui.decorateWith("appui", "standardEmrPage")
-    ui.includeCss("adminui", "account.css")
+
+    ui.includeJavascript("uicommons", "angular.min.js")
+    ui.includeJavascript("uicommons", "angular-resource.min.js")
+    ui.includeJavascript("uicommons", "angular-ui/angular-ui-router.min.js")
+    ui.includeJavascript("uicommons", "angular-app.js")
+    ui.includeJavascript("uicommons", "angular-common.js")
+    ui.includeJavascript("uicommons", "services/roleService.js")
+    ui.includeJavascript("uicommons", "services/privilegeService.js")
+    ui.includeJavascript("uicommons", "filters/display.js")
+    ui.includeJavascript("uicommons", "filters/serverDate.js")
+
+    ui.includeJavascript("uicommons", "ngDialog/ngDialog.js")
+    ui.includeCss("uicommons", "ngDialog/ngDialog.min.css")
+
+    ui.includeJavascript("adminui", "metadata/colTable.js")
+    ui.includeJavascript("adminui", "metadata/manageRoles.js")
 %>
+
 <script type="text/javascript">
     var breadcrumbs = [
-		{ icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
-		{ label: "${ ui.message('adminui.app.configureMetadata.label')}" , link: '${ui.pageLink("adminui", "metadata/configureMetadata")}'},
-		{ label: "${ ui.message("adminui.manageRoles.title")}" }
+        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+        { label: "${ ui.message('adminui.app.configureMetadata.label')}", link: '${ui.pageLink("adminui", "metadata/configureMetadata")}'},
+        { label: "${ ui.message("adminui.manageRoles.title")}" }
     ];
+    emr.loadMessages([
+        "uicommons.generalSavedNotification"
+    ]);
 </script>
 
-<% i=0 %>
-
-<h3>${ ui.message("adminui.manageRoles.title") }</h3>
-
-<input type="submit" class="button" value="${ui.message("adminui.addNewRole.label")}" onclick="window.location='/${ contextPath }/adminui/metadata/roles/role.page'"/>
+<style type="text/css">
+    /* TODO move this to a shared SCSS file */
+    tr.retired {
+        text-decoration: line-through;
+    }
 
-<hr>
-<table id="list-roles" cellspacing="0" cellpadding="2">
-	<thead>
-		<tr>
-			<th>${ ui.message("adminui.role.role") }</th>
-			<th>${ ui.message("adminui.role.inheritedRoles") }</th>
-			<th>${ ui.message("adminui.role.privileges") }</th>
-			<th></th>
-		</tr>
-	</thead>
-	<tbody>
-		<% roles.each{  %>
-	 	<tr>
-	 		<td>
-				 ${ ui.format(it.role) }
-			</td>
+    input.ng-touched.ng-invalid {
```

Comment:
These classes are already included in the uicommons module, just update your local copy of uicommons module and and it will magically happen. :)